crates/wasm/fs/private/forest.rs

```diff
@@ -23,4 +35,48 @@ impl PrivateForest {
     pub fn new() -> PrivateForest {
         Self(Rc::new(WnfsPrivateForest::default()))
     }
+
+    #[wasm_bindgen]
+    pub fn get(
+        &self,
+        saturated_namefilter_hash: Uint8Array,
+        revision_key: Uint8Array,
+        store: BlockStore,
+    ) -> JsResult<Promise> {
+        let store = ForeignBlockStore(store);
+        let forest = self.0.clone();
+
+        let saturated_name_hash = expect_bytes::<HASH_BYTE_SIZE>(saturated_namefilter_hash)?;
+
+        let key_bytes = expect_bytes::<KEY_BYTE_SIZE>(revision_key)?;
+        let key = Key::new(key_bytes);
+        let ratchet_key = RatchetKey(key);
+
+        let private_ref = PrivateRef::from_ratchet_key(saturated_name_hash, ratchet_key);
```

Comment:
Most likely another PR, but should we expose `PrivateRef` pointer to the user instead of constructing it from `RatchetKey` every time. And it conforms closely with the Rust API.

Reply:
@appcypher as then this should be from/try_from implementations right?

Reply:
Hm. Maybe. I'll keep this in mind when working on the JS side.